README.md

```diff
@@ -8,6 +8,7 @@ Gitops native package management
     the `profilesubscription`, `source` and `helm` controllers.
 
 1. Then subscribe to the example [nginx-profile](https://github.com/weaveworks/nginx-profile): `kubectl apply -f examples/profile-subscription.yaml`
+and deploy an example catalog `kubectl apply -f examples/profile-catalog-source.yaml`
```

Comment:
can we change the ordering of the steps here?

i think they should be something like:

1. deploy alll crds
2. create catalog
3. look at catalog
4. subscribe
5. profit

Reply:
done